tests/test_other.py

```diff
@@ -437,43 +437,55 @@ def test_emcc_5(self):
         self.assertContained('hello, world!', run_js('a.out.js'))
         assert test(open('a.out.js').read()), text
 
-  def test_emcc_6(self):
-    for compiler in [EMCC, EMXX]:
-      # Compiling two source files into a final JS.
-      for args, target in [([], 'a.out.js'), (['-o', 'combined.js'], 'combined.js')]:
-        self.clear()
-        run_process([PYTHON, compiler, path_from_root('tests', 'twopart_main.cpp'), path_from_root('tests', 'twopart_side.cpp')] + args)
-        self.assertContained('side got: hello from main, over', run_js(target))
+  def test_multiple_sources(self):
+    # Compiling two sources at a time should work.
+    cmd = [PYTHON, EMCC, '-c', path_from_root('tests', 'twopart_main.cpp'), path_from_root('tests', 'twopart_side.cpp')]
+    run_process(cmd)
 
-        # Compiling two files with -c will generate separate .bc files
-        self.clear()
-        expect_error = '-o' in args # specifying -o and -c is an error
-        cmd = [PYTHON, compiler, path_from_root('tests', 'twopart_main.cpp'), path_from_root('tests', 'twopart_side.cpp'), '-c'] + args
-        if expect_error:
-          err = self.expect_fail(cmd)
-          self.assertContained('fatal error', err)
-          continue
+    # Object files should be generated by default in the current working
```

Comment:
This feels like a separate test, it's not dependent on multiple sources (though it is a property that could be different for multiple sources?)

Reply:
Yeah, we have some handling in the driver that special cases single files, so I wanted to test this specifically here.